krun/platform.py

```diff
@@ -875,6 +877,25 @@ def _check_cset_shield(self):
             out, _, _ = run_shell_cmd(cmd)
             debug(out)  # cset is quite chatty on stdout
 
+    def _check_dmesg_unrestricted(self):
+        debug("Checking if dmesg buffer is restricted")
+
+        if os.path.exists(LinuxPlatform.RESTRICT_DMESG_FILE):
+            cmd = "%s cat %s" % (self.change_user_cmd,
+                                 LinuxPlatform.RESTRICT_DMESG_FILE)
+            out, _, _ = run_shell_cmd(cmd)
+            if int(out) == 0:
```

Comment:
These two lines might be simpler as `if int(run_shell_cmd(cmd)[0])) == 0`? [I'm also curious if we should catch the "file doesn't exist and / or returns something other than an integer" case...]

Reply:
The file must exist due to a earlier check. If it aint an int, an exception is raised, which is caught and emailed.